warehouse/redshift_test.go

```diff
@@ -146,10 +146,10 @@ func TestValidateSchemaConfig(t *testing.T) {
 		wh := NewRedshift(tc.conf)
 		err := wh.validateSchemaConfig()
 		if tc.hasError && err == nil {
-			t.Errorf("expected Redshift.validateSchemaConfig() to return an error when config.Config.Redshift.DatabaseSchema is empty")
+			t.Errorf("expected Redshift.validateSchemaConfig() to return an error when config.configuration.Redshift.DatabaseSchema is empty")
```

Comment:
Were changes in this file intentional?

Reply:
they have been reverted.  The change was actually because I changed the variable config to be private in an earlier version.

Reply:
You still need to remove these changes.

Reply:
done. 
```
$ git diff master -- warehouse/*.go
$
```